### DISCUSSION TITLE
id값을 String에서 Long으로 변환 위치에 관한 논의

### DISCUSSION THREAD

**Discussion by KwonYeKyeong** [slamapp/server#30] [General]
프론트와 협의 끝에 request / response에서는 id값을 String으로 하기로 했습니다.
그래서 저희 모두 코드에서 id값을 String에서 Long으로 변환하는 작업을 추가하게 되었는데 다들 어디서 이 작업을 수행하시는 지 궁금해졌습니다.
저는 현재 Controller단에서 이 작업을 수행하게끔 수정했습니다. (사진에서 48번째 줄)
<img width="965" alt="스크린샷 2022-03-03 오후 10 47 09" src="https://user-images.githubusercontent.com/65434196/156577175-bb9c5bcc-1aae-453e-b552-36eaeb2f75dd.png">
이 부분은 모두 공통적으로 수행하는 부분이라 통일했으면 좋겠는데 다들 어떠신가요??

**Comment by locodingve** [comment]
아하! 저의 도메인 (채팅이나 알람)은 많은 정보가 생성될 수 있어서, 그에 대한 이슈 발생 방지 목적으로 UUID를 적용할 예정입니다. (<TMI> 본 내용에 대해서는 2월에 논의되어서 확정된 부분입니다. 그리고 마누와도 공유가 된 부분입니다~)

그런데 젤리가 말씀하신 대로 통일하면 좋을 것 같아요! 
저는 서비스 내에서 또는 컨버터부분에서 변환되면 좋을 것 같아요~ 
이번주 토요일 날 좀더 자세하게 논의해봐요~

그리고 사용자 ID를 얻고자 할 때, 직접 함수를 사용해서 진행을 하고 있습니다. 
그런데 이 부분을 인터럽터를 사용해서, controller단까지 정보가 전달 될 수 있도록 처리할 수 있다고 합니다. (컨트롤러 단에서 어노테이션 사용하여, 사용자 ID 정보 추출가능)
이 내용도 토욜날 함께 논의해봐요~

**Comment by locodingve** [comment]
젤리 의견 굿굿!! 👍
좋은 의견 감사합니다.
